ecosystem/sep-0024.md

```diff
@@ -6,8 +6,8 @@ Title: Hosted Deposit and Withdrawal
 Author: SDF
 Status: Active
 Created: 2019-09-18
-Updated: 2021-08-17
-Version 1.4.0
+Updated: 2021-10-05
+Version 1.5.0
```

Comment:
I am not sure if 1.4 -> 1.5 indicates breaking change? 

Since these fields are optional, they should not be breaking.

Reply:
Only major version increments are considered "breaking" 👍